src/js/components/chat-input.jsx

```diff
@@ -86,9 +86,14 @@ export class ChatInputComponent extends Component {
         let sendButton;
 
         const buttonClassNames = ['send'];
+        const buttonStyle = {};
 
         if (this.state.text.trim()) {
             buttonClassNames.push('active');
+
+            if (this.props.settings.accentColor) {
+                buttonStyle.color = `#${this.props.settings.accentColor}`;
```

Comment:
haha no, this code is only the send button which uses `color`, the action buttons use `background-color` in `src/js/components/action.jsx`